feat: INSTR function #881 (#5385)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/scalar-functions.md

Capitalize the first letter in each word and convert all other letters
to lowercase. Words are delimited by whitespace.

INSTR
-----

`INSTR(string, substring, [position], [occurrence])`

> Returns a position of `substring` in `string` (first character is at position 1).

Reviewer: ```suggestion
Returns the position of `substring` in `string`. The first character is at position 1.
```

> If `position` is given search starts from that position. 

Reviewer 2: we should specify here what it means to have negative position

> If `position` is given search starts from that position. 

Reviewer: ```suggestion
If `position` is provided, search starts from the specified position. 
```

> If `occurrence` is given position of n-th occurrence is returned.

Reviewer: ```suggestion
If `occurrence` is provided, the position of *n*-th occurrence is returned.
```

Negative `position` causes the search to work from end to start of `string`.

> If `substring` is not found the return value is 0.

Reviewer: ```suggestion
If `substring` is not found, the return value is 0.
```
